features/step_definitions/framework_application_steps.rb

```diff
@@ -149,6 +153,7 @@
 
   new_service = get_draft_service_copied_from(@existing_service, @framework['slug'])
   @new_service_href = "/suppliers/frameworks/#{@framework['slug']}/submissions/#{new_service['lotSlug']}/#{new_service['id']}"
+  puts "Draft service URL: #{@new_service_href}"
```

Comment:
Does this need to be here?

Reply:
Oh, I put it in to trace down a bug, I thought it might be useful for future; they only show up if the test fails anyway.

Reply:
It's just been useful - thanks! 🙂